src/main/java/DeadlineTasks.java

```diff
@@ -0,0 +1,19 @@
+public class DeadlineTasks extends ShowTasks{
+
+    ShowTasks showTasks = new ShowTasks();
+
+    public DeadlineTasks(){
+
+    }
+
+
+    public void deadLineTask(String deadLine){
```

Comment:
Shall we use verbs for methods/functions? The purpose of this method is not so straightforward to me. I am not sure what is this "deadLineTask" handling.  Shall we consider to edit the name and use the name to explain?